cmd/cpe2cve/cpe2cve.go

```diff
@@ -300,9 +346,10 @@ func Main() int {
 		pprof.StartCPUProfile(f)
 		defer pprof.StopCPUProfile()
 	}
-
+	if len(cfg.HttpBindAt) > 0 {
+		go httpServer(caches, cfg)
```

Comment:
This seems a bit leaky. Also this would make running it as a daemon or even simply in background complicated/impossible. For most use cases I can imagine, it would be better to block and terminate here. E.g. (requires chaning httpServer signature a bit):
```
if err := httpServer(caches, cfg); err != nil {
    flog.Fatal(err)
}
return 0
```